llvmlite/binding/targets.py

```diff
@@ -6,10 +6,28 @@
                     c_void_p, string_at, byref)
 
 from . import ffi
-from .module import parse_assembly
 from .common import _decode_string, _encode_string
 
 
+def get_process_triple():
+    """
+    Returns the target triple of the process when using jit for the current
+    process.
+    """
+    with ffi.OutputString() as out:
+        ffi.lib.LLVMPY_GetProcessTriple(out)
+        return str(out)
+
+def get_host_cpu_features():
+    """
+    Returns a string (e.g. "+sse,-avx) indicating the available CPU features.
+    May return an empty string if LLVM cannot detect the CPU features.
+    """
+    with ffi.OutputString() as out:
+        ffi.lib.LLVMPY_GetHostCPUFeatures(out)
+        return str(out)
```

Comment:
I am lazy and the output can go directly into the feature attributes of `create_target_machine` https://github.com/numba/llvmlite/blob/93a41b1243b6f3fe5b751ba842ca01794f7be6da/llvmlite/binding/targets.py#L130
=)

Perhaps, wrapping the `features` string in an object for providing the dictionary lookup lazily would be nice.